src/main/java/org/tc/mail/MailNotificationSender.java

```diff
@@ -42,15 +49,15 @@ public void sendManagerNotification(Course course) {
         try {
             MimeMessageHelper helper = new MimeMessageHelper(message, true);
             helper.setSubject(COURSE_ANNOUNCEMENT_SUBJECT);
-            String[] emailsTo = {roleService.getDepartmentManager().getEmail(),
-                    roleService.getKnowledgeManager().getEmail()};
+            String[] emailsTo = {roleService.getDepartmentManager().get().getEmail(),
```

Comment:
I thought about it, I thought that I can set OrElse() and return default value here, but there are preset knowledge and department managers, so in this app **won't** happen NPE here